### ISSUE TITLE
Support for Laravel

### ISSUE THREAD

**Issue by akoSalman** [phpmentors-jp/workflower#37]
Is there any package like this for Laravel framework?

**Comment by iteman** [comment]
Yes. I think Workflower can be used with Laravel since it only uses Symfony components, not Symfony framework. But you may want well-integrated features such as WorkflowerBundle for Symfony.

**Comment by lakilevi** [comment]
Did you manage to do this? I need the same thing now, to integrate workflower into Laravel. Could you give me some hints?

**Comment by akoSalman** [comment]
@lakilevi No, I didn't use it.

**Comment by lakilevi** [comment]
Thanks!

**Comment by unpluggedweb** [comment]
What did you use instead for Laravel?

**Comment by akoSalman** [comment]
I handled it in some tricky way using roles and permissions like this:

**Each user with a specific `role`,  `can(permission)` do some specific `actions` on our object in a specific `state`.**
